### DISCUSSION TITLE
Different resource requests and limits for registration-only runners

### DISCUSSION THREAD

**Discussion by Puneeth-n** [actions/actions-runner-controller#624] [Ideas]
Hi, I was wondering if having different resource requests and limits for registration-only runners would make sense when scaling from 0.

The reason being, I use `dockerVolumeMounts` and pin each runner to a single instance. I also use node selector to run the runners on a different size of instances. It would be nice to probably not also have the single instance running when not needed.

**Comment by toast-gear** [comment]
Almost certainly however it's probably not worth doing at this point unless someone really wants to do it. GitHub in theory are changing their backend service very soon (this month) to allow scaling from 0 without the need for fake runners, see https://github.com/actions-runner-controller/actions-runner-controller/issues/470 for details

**Reply by Puneeth-n** [reply]
nice 👍🏾

**Comment by toast-gear** [comment]
Registration runners have been removed https://github.com/actions-runner-controller/actions-runner-controller/issues/859 as they are no longer needed for GHES => 3.3.0 or Cloud.
